kafka/util.go

```diff
@@ -22,20 +22,40 @@ func parseAddrs(u *url.URL) []string {
 	return []string{addr}
 }
 
+func isTrueValue(v string) bool {
+	return !(v == "" || v[0] == '0' || v[0] == 'f' || v[0] == 'F' || v[0] == 'n' || v[0] == 'N')
```

Comment:
Nit, but too much flexibility is not very cool, IMO. Just documenting `{true,false}` would be enough.

Reply:
see, I prefer `=1` flags on urls ;)